bundles/org.eclipse.swt/Eclipse SWT Custom Widgets/common/org/eclipse/swt/custom/CCombo.java

```diff
@@ -2026,19 +2027,24 @@ public boolean traverse(int event){
 	return super.traverse(event);
 }
 
+private void handleDPIChange(Event event) {
+	updateAndRefreshChildren(childWidget -> {
+		childWidget.notifyListeners(SWT.ZoomChanged, event);
+	});
```

Comment:
I find this split into two methods with an additional lambda quite hard to read. Why not just ... ?
```suggestion
	text.notifyListeners(SWT.ZoomChanged, event);
	list.notifyListeners(SWT.ZoomChanged, event);
	arrow.notifyListeners(SWT.ZoomChanged, event);
	popup.notifyListeners(SWT.ZoomChanged, event);
```

Reply:
Yes I missed that one out. Adapting.